praas-app-ionic/src/api/praas.ts

```diff
@@ -128,11 +128,17 @@ const praas = {
         body: JSON.stringify(data),
       });
     },
-    update(data: any) {
-      const cid = data.conduit.id;
+    update(conduit: any) {
+      debugger;
+      const cid = conduit.id;
```

Comment:
I'd instead prefer to destructure it.
And regarding naming conventions, I think it is okay to have short names than a longer explicit name.
like `conduits.map((c) => c.id)` would be fine as long as we are not doing any bigger processing on the variables.

Reply:
@sirajulm at the very end we write code which should be maintainable. And this code is going through a build and bundling process which anyways optimises it. So my arguments are against it. Always name the variables right whether it be to think it in the context or not in the context. This leaves less chances of unreadable code. https://3.basecamp.com/4109673/buckets/9419490/messages/1654323701 please see to this on what the conventions to follow are

Reply:
I'm of the opinion that using shorter variables are okay, as long as the code is self explained. There is no meaning in writing or trying to spend time to figure out a proper variable name, as long as you dont do any manipulation on it or reuse it elsewhere.
Conventions are needed for maintainable code, and we need to use them as much possible, but thinking of too much convention always even for self explanatory code is just bike shedding 😄 
Anyway I've converted to use destructuring instead of creating seperate variable already 😛.